mem0/memory/main.py

```diff
@@ -1404,15 +1407,18 @@ async def history(self, memory_id):
         capture_event("mem0.history", self, {"memory_id": memory_id, "sync_type": "async"})
         return await asyncio.to_thread(self.db.get_history, memory_id)
 
-    async def _create_memory(self, data, existing_embeddings, metadata=None):
+    async def _create_memory(self, data, existing_embeddings, metadata: dict | None = None):
```

Comment:
Change this to `Optional[dict]` because this project supports Python >=3.9 and `|` was introduced in Python 3.10